doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1154 @@
+       "      <td>/Users/christy/Documents/github_ray_temp/ray/d...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   location_id                          model_name       error  \\\n",
+       "0          152  DecisionTreeRegressor(max_depth=3)  224.300000   \n",
+       "1          166  DecisionTreeRegressor(max_depth=3)  227.661902   \n",
+       "2          145  DecisionTreeRegressor(max_depth=3)  461.833333   \n",
+       "\n",
+       "                                          checkpoint  \\\n",
+       "0  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "1  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "2  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "\n",
+       "                                      checkpoint_dir  \n",
+       "0  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "1  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "2  /Users/christy/Documents/github_ray_temp/ray/d...  "
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Keep only 1 model per location_id with minimum error\n",
+    "final_df = results_df.reset_index()\n",
+    "final_df = final_df.loc[final_df.groupby('location_id')['error'].idxmin()].copy()\n",
+    "final_df.sort_values(by=[\"error\"], inplace=True)\n",
+    "final_df.reset_index(inplace=True, drop=True)\n",
+    "print(final_df.dtypes)\n",
+    "final_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4554830-1546-4353-a41e-621bdf46aeab",
+   "metadata": {},
+   "source": [
+    "## Load a model from checkpoint and perform inference  <a class=\"anchor\" id=\"load_checkpoint\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "6f5c78fa-38a9-4e8c-bddc-ffbb45aa2d85",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/christy/Documents/github_ray_temp/ray/doc/source/ray-air/examples/my_Tune_logs/batch_tuning/train_model_39d58_00005_5_location=152,model=DecisionTreeRegressor_max_depth_3_2022-11-08_01-32-15/checkpoint_000000\n",
+      "<class 'ray.train.sklearn.sklearn_checkpoint.SklearnCheckpoint'>\n",
+      "<class 'ray.train.sklearn.sklearn_predictor.SklearnPredictor'>\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get a checkpoint_dir.\n",
+    "checkpoint_path = final_df.checkpoint_dir[0]\n",
+    "print(checkpoint_path)\n",
+    "\n",
+    "# In another function or script, recover Checkpoint object from path\n",
+    "checkpoint = SklearnCheckpoint.from_directory(checkpoint_path)\n",
+    "print(type(checkpoint))\n",
+    "\n",
+    "# Restore a predictor object from checkpoint\n",
+    "predictor = SklearnPredictor.from_checkpoint(checkpoint)\n",
```

Comment:
AIR Predictors make batch inference easy, but for this example, we can just use the sklearn model directly to make it simpler.